packages/devtools_app/lib/src/flutter/common_widgets.dart

```diff
@@ -278,6 +278,30 @@ Widget exitOfflineButton(FutureOr<void> Function() onPressed) {
   );
 }
 
+/// Toggle button for use as a child of a [ToggleButtons] widget.
+Widget devToolsToggleButton({
```

Comment:
maybe make this a Widget instead of a method? It looks a lot like a widget.
There are a bunch of other methods in this file that should also be widgets.
Making them widgets has marginal performance benefits and helps tools like the Inspector do a better job as the widget tree is more semantic and you have the right creation locations. Maybe add a TODO on the other methods in this class that should be widgets but aren't.